packages/modules/viewer-react/src/services/RpcInitializer.ts

```diff
@@ -0,0 +1,95 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import type {
+  BentleyCloudRpcParams,
+  RpcInterface,
+  RpcInterfaceDefinition,
+} from "@itwin/core-common";
+import {
+  BentleyCloudRpcManager,
+  IModelReadRpcInterface,
+  IModelTileRpcInterface,
+} from "@itwin/core-common";
+import { PresentationRpcInterface } from "@itwin/presentation-common";
+
+import type { BackendConfiguration } from "../types";
+
```

Comment:
as per the discussion we had on the side, i think all this needs to move to the web-viewer pkg.
types, and the class. i dont think we need it to be a static at all.